packages/ses/package.json

```diff
@@ -46,7 +46,7 @@
     "rollup-plugin-terser": "^5.1.3",
     "sinon": "8.0.4",
     "tap": "14.10.5",
-    "tape": "4.12.1"
+    "tape": "5.0.1"
```

Comment:
According to https://github.com/Agoric/SES-shim/issues/293 (attn @warner ) tape 4 is compat with SES but tape 5 is not.

Reply:
That’s unfortunate since the tests behave correctly under version 5 and we’ll have to hack the plan counts to make them work under 4. I’ll remove the tape upgrade and see whether I can make the tests work on both versions.